### PR TITLE
feat: support alias in matcher define_enum

### PR DESCRIPTION
**Pull request by brunohkbx**

### CODE REVIEW

**Comment by wenderjean** [comment]
Do we have an issue related? :)

**Comment by brunohkbx** [comment]
Do we need one? I ran into this when I was creating some aliases for a legacy table in a database.

**Comment by wenderjean** [comment]
It would be good to keep tracking and also describe what you're solving. It could be both a bugfix and a new feature.

**Comment by brunohkbx** [comment]
Makes sense to me. @mcmire thoughts? I can describe it on an issue if you want.

**Comment by mcmire** [comment]
This is fine. I appreciate the updates to the documentation because I could figure out pretty quickly what this was about. No need for an issue :)

**Review by mcmire — COMMENTED** [review]
One little thing, but this makes sense otherwise! I like how simple this PR is.

File: lib/shoulda/matchers/active_record/define_enum_for_matcher.rb [hunk 1]
```diff
@@ -370,7 +376,10 @@ def expected_column_type
         end
 
         def column
-          model.columns_hash[attribute_name.to_s]
+          key = attribute_name.to_s
+          column_name = model.attribute_aliases[key] || key
```
[the remark this answers is not in the document]
Comment (brunohkbx) [reply lib/shoulda/matchers/active_record/define_enum_for_matcher.rb]: I updated it

**Comment by brunohkbx** [comment]
@mcmire would you mind reviewing it again? :smiley:

**Comment by mcmire** [comment]
Sorry for the delay. I was waiting until I had confirmation on the plan for the next version, but it seems like we've got that settled now. Looks good, merging now!

**Pull request merged**